### ISSUE TITLE
Resolve CDF warnings

### ISSUE THREAD

**Issue by pacesm** [ESA-VirES/VirES-Server#161]
```
spacepy/time.py:2294: UserWarning: Leapseconds may be out of date. Use spacepy.toolbox.update(leapsecs=True)
  warnings.warn('Leapseconds may be out of date.'

spacepy/pycdf/__init__.py:1953: DeprecationWarning: spacepy.pycdf.lib.set_backward not called; making backward-compatible CDF. This default will change in the future.
```

**Comment by pacesm** [comment]
The second `pycdf` deprecation warning is fixed in 4915ee7c4ffd8cc3e240e2f4f0bb9b6eca5ec1d3

The warning about the updated leap-seconds table does not get resolved by the recommended update. It seems like the SW is confused by the fact that there has not been any new leap second added since 2016-07-08.

**Comment by pacesm** [comment]
To suppress warnings from `spacepy` from Python code run
```python
 warnings.filterwarnings("ignore", category=UserWarning, module="spacepy")
```
or alternatively disable warnings via the environmental variable
```sh
export PYTHONWARNINGS="ignore"
```
